### ISSUE TITLE
Heading elements are not in a sequentially-descending order

### ISSUE THREAD

**Issue by var-bin** [Varbintech/next.varbintechcom#47]
Properly ordered headings that do not skip levels convey the semantic structure of the page, making it easier to navigate and understand when using assistive technologies. [Learn more about heading order](https://dequeuniversity.com/rules/axe/4.6/heading-order).

Failing Elements
--
Navigation `<h6 class="MuiTypography-root MuiTypography-subtitle2 mui-style-1xw12my">`

---

![Image](https://user-images.githubusercontent.com/3194163/229099647-b82a44cf-230e-416a-95ee-98972af5d28b.jpg)

## How to fix it:
- Add `component` property. Example:
```tsx
<Typography
  variant="subtitle1"
  align="center"
  component="h2" // <-- Here it is, but we should use h3 or h4
  sx={{
    margin: '0 auto 32px',
    maxWidth: '620px',
  }}
>
  We help startups and companies of all shapes and sizes to build high-quality Front-End
  solutions.
</Typography>
```
